### PR TITLE
Re-arrange AWS cluster creation form properties, hide some

### PR DESCRIPTION
**Pull request by marians**
### What does this PR do?

This PR adds configuration to improve the AWS CAPI cluster creation form by

- Hiding unwanted properties
- Sorting properties
- Customizing certain properties (instance type, cluster name)

### What is the effect of this change to users?

See below

### How does it look like?

image

### Any background context you can provide?

Issue https://github.com/giantswarm/roadmap/issues/1181

### Do the docs need to be updated?

No

### Should this change be mentioned in the release notes?

No

### CODE REVIEW

File: src/components/MAPI/clusters/CreateClusterAppBundles/schemaUtils.ts [hunk 1]
```diff
@@ -53,24 +53,72 @@ const formPropsProviderCAPA: Record<string, FormPropsPartial> = {
...
+        'ui:widget': 'hidden',
+      },
+      managementCluster: {
+        'ui:widget': 'hidden',
+      },
+      metadata: {
+        'ui:order': ['name', 'description', '*'],
+        name: {
+          // TODO: fix crash
+          //'ui:widget': ClusterNameWidget,
```
Comment (marians) [inline src/components/MAPI/clusters/CreateClusterAppBundles/schemaUtils.ts:109]: @gusevda Can you please help me with this one? If I uncomment line 109, the app crashes ("can't read property length of undefined", if I remember correctly).

Comment (gusevda) [reply src/components/MAPI/clusters/CreateClusterAppBundles/schemaUtils.ts:109]: you should also change `formData` property below to match the new structure. It should be
```
formData: (clusterName, organization) => {
    return {
        metadata: {  
            name: clusterName,
            organization,
        }
    };
},
```

Comment (marians) [reply src/components/MAPI/clusters/CreateClusterAppBundles/schemaUtils.ts:109]: Thanks, that worked.

**Review by kuosandys — APPROVED** [review]
Looks great!

**Pull request merged**